src/main/mdo/rule.mdo

```diff
@@ -227,5 +237,41 @@ under the License.
         </codeSegment>
       </codeSegments>
     </class>
+    <class xsd.compositor="sequence">
+      <name>VersionComparator</name>
+      <version>2.0+</version>
```

Comment:
This mean this breaks compatibility so we need to make change in the version (3.X*) at least...

Reply:
It does?  Existing rule sets should still work because versionComparator is not marked as required.

Reply:
@khmarbaise I think I answered my own question about compatibility.  I was thinking about schema compatibility when I asked my question, but you were referring to binary compatibility.  Can you confirm/deny/explain?